SteamReview/SteamReview/Steam.cpp

```diff
@@ -226,19 +246,31 @@ bool FSteam::AddNewGameToCategory(const FGame InGame)
 
 			int CategoryChosen;
 			std::cin >> CategoryChosen;
-			CategoryChosen = ValidateInput(CategoryChosen, 1, CategoryContainer.GetCurrentCategoryAmount()) - 1;
+			ValidateInput(CategoryChosen, 1, CategoryContainer.GetCurrentCategoryAmount());
+			
+			//Subtract 1 to select the current object inside of the array
+			CategoryChosen = CategoryChosen - 1;
 
 			//adding the game to category
-			if (CategoryContainer.AddGameToCategory(CategoryChosen, InGame))
+			FCategory Category;
+			if(CategoryContainer.GetCategory(CategoryChosen, Category))
 			{
-				std::cout << "------------------------------------------\n" << std::endl;
-				std::cout << "The game is going to add to the category: " << CategoryContainer.GetCategory(CategoryChosen - 1).GetCategoryName() << std::endl;
-				return true;
+				if (CategoryContainer.AddGameToCategory(CategoryChosen, InGame))
+				{
+					std::cout << "------------------------------------------\n" << std::endl;
+					std::cout << "The game is going to add to the category: " << Category.GetCategoryName() << std::endl;
```

Comment:
The game isn't adding anything to the category, it is instead going to be added. It has in fact, by this moment, already been added, so 'The game has been to the category' would make much more sense as a message for the user